src/utils/getSigner.ts

```diff
@@ -0,0 +1,141 @@
+import type { ethers } from 'ethers';
+import { KJUR } from 'jsrsasign';
+import { createConnectWalletButton } from './CheckoutWithEth/createConnectWalletButton';
+import { createWalletConnectorContainer } from './CheckoutWithEth/createWalletConnectorContainer';
+import { importWagmiLibraries } from './CheckoutWithEth/importWagmi';
+import { isTestnet } from './CheckoutWithEth/isTestnet';
+import { mapWalletConnectorNameToPaperWalletName } from './CheckoutWithEth/mapWalletConnectorToPaperWalletName';
+import { setUpSigner } from './CheckoutWithEth/setUpSigner';
+
+export async function getSignerInfo({
+  container,
+  sdkClientSecret,
+  appName,
+}: {
+  container: HTMLElement;
+  sdkClientSecret: string;
+  appName?: string;
+}) {
+  try {
+    const {
+      fetchSigner,
+      configureChains,
+      connect,
+      createClient,
+      goerli,
+      mainnet,
+      MetaMaskConnector,
+      WalletConnectLegacyConnector,
+      CoinbaseWalletConnector,
+    } = await importWagmiLibraries();
```

Comment:
Seems inconsistent to have a helper func here to import a bunch of stuff. Maybe just inline these imports here to avoid jumping to another file, and this import is only used once.